dev-lang/janet/janet-1.12.2.ebuild

```diff
@@ -12,10 +12,12 @@ SRC_URI="https://github.com/janet-lang/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.g
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="static-libs"
+IUSE="+static-libs"
```

Comment:
Why is this enabled by default? We're trying to get rid of `static-libs`, not encourage using it.

Reply:
Because without this `jpm` breaks. `jpm build` creates a static executable which can be deployed anywhere.
I can remove this and add `elog` message to notify the users that if this feature is desired they should manually enable it.
WDYT?

Reply:
OK. static-libs is no longer the default.